docs/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ title: Changelog
 
 [Full Changelog](https://github.com/scalameta/scalafmt/compare/v2.4.1...v2.4.2)
 
+Hello
+worl
```

Comment:
1. typo
2. should it be here?

Reply:
never mind, disappeared.

Reply:
I used this to test that the CI would fail